test/acceptance/features/examples.feature

```diff
@@ -53,3 +54,72 @@ Feature: Verify examples provided in Service Binding Operator github repository
         And Secret "sbr-to-bind-hello-app-to-route" contains "ROUTE_HOST" key with value "example-sbo.apps.ci-ln-smyggvb-d5d6b.origin-ci-int-aws.dev.rhcloud.com"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/pod_spec_path
+    Scenario: Testing PodSpec path
+        Given DB "db-pod-spec-path-ex" is running
+        * The Custom Resource Definition is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: appconfigs.stable.example.com
+            spec:
+                group: stable.example.com
+                versions:
+                  - name: v1
+                    served: true
+                    storage: true
+                scope: Namespaced
+                names:
+                    plural: appconfigs
+                    singular: appconfig
+                    kind: AppConfig
+                    shortNames:
+                    - ac
+            """
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: AppConfig
+            metadata:
+                name: appconfig-pod-spec-path-ex
+            spec:
+                uri: "some uri"
+                Command: "some command"
+                image: my-image
+                spec:
+                    containers:
+                    - name: hello-world
+                      # Image from dockerhub, This is the import path for the Go binary to build and run.
+                      image: yusufkaratoprak/kubernetes-gosample:latest
+                      ports:
+                      - containerPort: 8090
+            """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-pod-spec-path-ex
+            spec:
+                envVarPrefix: qiye111
+                application:
+                    name: appconfig-pod-spec-path-ex
+                    group: stable.example.com
+                    version: v1
+                    resource: appconfigs
+                    bindingPath:
+                        containersPath: spec.spec.containers
+                services:
+                  - group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-pod-spec-path-ex
+                    id: zzz
+                    envVarPrefix: qiye
+            """
+        Then Service Binding "binding-request-pod-spec-path-ex" is ready
+        And Secret "binding-request-pod-spec-path-ex" has been injected in to CR "appconfig-pod-spec-path-ex" of kind "AppConfig" at path "{.spec.spec.containers[0].envFrom[0].secretRef.name}"
+        When Service binding "binding-request-pod-spec-path-ex" is deleted
```

Comment:
each scenario should have only one `When` block, otherwise it means that we are testing multiple things which is not good.

Reply:
In this scenario, we're testing two things, 1. Binding 2. Unbinding, that's why there are two whens.

Reply:
fine, but example can contain multiple scenario in it.. Hence, it would be good to split this into multiple scenarios.

Reply:
@pmacik WDYT?

Reply:
@pratikjagrut I agree with @pedjak since the example demonstrate 2 things - binding and unbinding - it should have multiple scenarios, one for each thing. Of course that means repeating a set of initial steps.